### DISCUSSION TITLE
How can I set query parameters in BingConnector?

### DISCUSSION THREAD

**Discussion by takeo-iw** [microsoft/semantic-kernel#2545] [Q&A] [answered]
I want to set query parameters, e.g. count, offset, cc(ja-JP) in WebSearchEngineSkill(BingConnector).
When I ask in Japanese, "What's the weather like in xxx city today?", I get incorrect result at all. It's correct in English.
Does the current version only support English?

I tried notebooks 10-BingSearch-using-kernel as below. It's no weather returned.

![image](https://github.com/microsoft/semantic-kernel/assets/108264521/0b5c9ba6-4fb8-4c74-a9c0-da4a30f8a0e3)

English question is correct.

![image](https://github.com/microsoft/semantic-kernel/assets/108264521/fcd54eb0-aaaa-42ef-8de7-a57dd2e0ae38)

**Comment by moonbox3** [comment, marked as answer]
@takeo-iw we do have some work items around improving the functionality of the BingSearch connector. It isn't the highest priority right now as we move towards v1, but we will get there. Feel free to propose a PR if you'd like. Thanks.
